llm/yamls/mosaic_gpt/760m.yaml

```diff
@@ -82,6 +89,7 @@ grad_clip_norm: 1.0
 
 # System
 seed: 17
+device_eval_batch_size: 4
 device_train_microbatch_size: 4
```

Comment:
If the 1b model support `device batch_size = 16`, then 760m model should also be set to at least 16.

Reply:
Hmm i was trying to play it safe and just put values that will guaranteed not OOM... so basically if`device_train_microbatch_size` succeeds in train mode then it should also succeed in eval mode.... I think I'm gonna leave it as is for now just because I don't have the bandwidth to profile the true ceiling eval_batch_sizes that work for each config